repo/blob/azure/azure_storage.go

```diff
@@ -0,0 +1,273 @@
+// Package azure implements Azure Blob Storage.
+package azure
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/efarrer/iothrottler"
+	"github.com/pkg/errors"
+	"gocloud.dev/gcerrors"
+
+	"github.com/kopia/kopia/internal/retry"
+	"github.com/kopia/kopia/repo/blob"
+
+	"github.com/Azure/azure-storage-blob-go/azblob"
```

Comment:
nit: these imports should go in the 2nd group above.

Reply:
Sure. I am yet to do cleanup and refactoring